keyserver/handler.go

```diff
@@ -47,7 +48,15 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 	server.tbs[reg.Username] = tb
 	server.Unlock()
 
-	// TODO: store the user policies into DB after kv branch is merged
+	// store the user policies into DB
```

Comment:
If we're making the user policies part of the data blob stored into the DB, as we've been discussing in #46, I don't think we should be storing the user policies into the DB separately here, too.